.stylelintrc.yaml

```diff
@@ -0,0 +1,256 @@
+
+    # Other
+
+    - border-collapse
+    - border-spacing
+    - box-shadow
+    - caption-side
+    - content
+    - cursor
+    - empty-cells
+    - opacity
+    - overflow
+    - quotes
+    - speak
+    - table-layout
+    - vertical-align
+    - visibility
+    - z-index
+  # The following prefix rules are enabled since we use autoprefixer
+  at-rule-no-vendor-prefix: true
+  media-feature-name-no-vendor-prefix: true
+  selector-no-vendor-prefix: true
+  value-no-vendor-prefix: true
+  # Usually if you're nesting past 3 levels deep there's a problem
+  max-nesting-depth: 3
+  # Because we adhere to BEM we can limit the amount of necessary compound selectors. Most should
+  # just be 1 level selector. However, modifiers can introduce an additional compound selector.
+  # Futhermore, generic qualifying selectors (e.g. `[dir="rtl"]`) can introduce yet another level.
+  selector-max-compound-selectors: 3
+  # For specificity: disallow IDs (0). Allow for complex combinations of classes, pseudo-classes,
+  # attribute modifiers based on selector-max-compound-selectors, plus an addition for
+  # pseudo-classes (4). Allow for pseudo-elements (1).
+  selector-max-specificity: 0,4,1
+  # Disallow "@extend" in scss.
+  # http://csswizardry.com/2016/02/mixins-better-for-performance/
+  # http://vanseodesign.com/css/sass-mixin-or-extend/
+  # Besides performance, @extend actually *changes* the selector precedence by creating a compound
+  # selector, which can lead to ambiguous results.
+  at-rule-blacklist:
+    - extend
+  # Extremely useful for typos, and anything emergent can be ignored by this rule
+  property-no-unknown: true
+  # There is no reason that a specific ID would be needed for UI components
+  selector-no-id: true
+  # Qualifying types are not needed when using a naming system like BEM
+  selector-no-qualifying-type: true
+  # In general, we should *never* be modifying elements within our components, since we can't
+  # predict the use cases in which users would add a certain type of element into a component.
+  # An exception to this may be in packages/material-design-lite, in which case this rule could be
+  # disabled for that file, with an explanation.
+  selector-no-type: true
+  # Styles for components should never need the universal selector.
+  selector-no-universal: true
+  # Ensure any defined symbols are prefixed with "mdl-"
+  # TODO: Remove "md-" pattern.
+  custom-media-pattern: ^mdl?-.+
+  custom-property-pattern: ^mdl?-.+
+  selector-class-pattern:
+    - ^mdl?-.+
+    -
+      resolveNestedSelectors: true
+  selector-id-pattern: ^mdl?-.+
+  # Names are more semantic than numbers
+  font-weight-notation: named-where-possible
+  # :root is an important enough selector that it always stands on it's own. Also, there is no
+  # reason to make root a compound selector (?)
+  selector-root-no-composition: true
+  # http://www.paulirish.com/2010/the-protocol-relative-url/
+  function-url-no-scheme-relative: true
```

Comment:
AH, OK. I was searching around and saw the initial issue. They were saying relative URLs there were good and should be enforced, so I thought this was doing that instead.